### ISSUE TITLE
Elevate constructor of JsonRpcMultiServer for customized ObjectMapper

### ISSUE THREAD

**Issue by mikelue** [briandilley/jsonrpc4j#130]
Is it possible to elevate
```java
private JsonRpcMultiServer(ObjectMapper mapper)
````
to
```java
public JsonRpcMultiServer(ObjectMapper mapper)
````

So that the customized `ObjectMapper` could be used in `JsonRpcMultiServer`.

Thanks

**Comment by gaborbernat** [comment]
It is, create a PR for it and I'll accept it.

**Comment by gaborbernat** [comment]
released with 1.4.4.
